CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -1915,7 +1915,7 @@ - (BOOL)hasSubscriptionAttributeValue:(NSString*)attributeId value:(NSString*)va
 }
 
 #pragma mark - Retrieving all the available tags from the channelConfig
-- (NSArray*)getAvailableTags {
+- (NSArray<NSString*>*)getAvailableTags {
```

Comment:
I think this method does not return an array. Please check again, I think it's `CPChannelTag*`
Also, please modify here:
CPChannelTag